src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -21,6 +21,17 @@ class Webui::WebuiController < ActionController::Base
   before_action :current_announcement
   after_action :clean_cache
 
+  unless Rails.env.production?
```

Comment:
huh?

Reply:
This is just for the gem I'm testing, the commit will be dropped